src/linker/specifications/environment_spark.yaml

```diff
@@ -5,13 +5,12 @@ slurm:
   partition: all.q
 # TODO: Move implementation resources (pipeline.yaml? input spec?)
 implementation_resources:
-  memory: 10  # GB
-  cpus: 2
+  memory: 1  # GB
+  cpus: 1
   time_limit: 1  # hours
 spark:
   workers:
     num_workers: 2  # num_workers + 1 nodes will be requested
-    cpus_per_task: 5
-    mem_per_cpu: 8  # GB
+    cpus_per_node: 1
+    mem_per_node: 1  # GB
```

Comment:
Request from @zmbc . He found that it's far easier to think about it on a per-node basis than trying to math out the resources per task

Reply:
There are two different changes here. `cpus_per_task` changed to `cpus_per_node` because (a) we no longer use multiple tasks since we are using an array job and (b) "task" is a Slurm specific term anyway whereas "node" is also the Spark term.

`mem_per_cpu` changed to `mem_per_node` because as Steve said, in my case study work I have found that the amount of memory you need does not scale at all linearly with CPUs.